gateway_common_test.go

```diff
@@ -0,0 +1,33 @@
+package gateway
+
+import (
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestDefaultInterface(t *testing.T) {
+	routeFile, err := filepath.Abs("./testdata/linux_route")
```

Comment:
Please embed testdata/linux_route inside this go file as a multi-line string. I'd prefer that over file i/o.

Reply:
I removed the test, I realized I had missed one that was already there that did the same thing